content/docs/reference/reproducibility.md

```diff
@@ -0,0 +1,46 @@
++++
+title="Reproducible Builds"
+weight=9
++++
+
+## Summary
+The Cloud Native Buildpacks project aims to create "Reproducible Builds" of container images. For image creation commands (`create-builder`, `package-buildpack`, `build`) `pack` creates container images in a reproducible fashion. "Reproducible" is hard to define but we'll do so by example and with a few caveats:
+
+## Examples
+---
+Running `pack build sample-hello-moon:test` multiple times produces a container image with the same image ID (*local* case)
+
+**Given**:
+- A workspace directory containing the same source code
+- The same builder image
+- The same set of buildpacks (see caveat below).
```

Comment:
Does the workspace directory only contain the source code or also the builder image and the buildpacks?
If the later is true, maybe it's better to write:
```
A workspace directory containing:
- The same source code
- The same builder image
- The same set of buildpacks
```

Reply:
I'm using workspace sort of vaguely here. This should just really be `- The same source code`